polling_stations/apps/data_importers/management/commands/import_mid_devon.py

```diff
@@ -1,48 +1,37 @@
-from data_importers.management.commands import BaseXpressDemocracyClubCsvImporter
+from data_importers.management.commands import BaseXpressWebLookupCsvImporter
 
 
-class Command(BaseXpressDemocracyClubCsvImporter):
+class Command(BaseXpressWebLookupCsvImporter):
     council_id = "MDE"
-    addresses_name = (
-        "2023-05-04/2023-03-17T16:54:00.659878/Democracy_Club__04May2023.tsv"
-    )
-    stations_name = (
-        "2023-05-04/2023-03-17T16:54:00.659878/Democracy_Club__04May2023.tsv"
-    )
-    elections = ["2023-05-04"]
+    addresses_name = "2024-05-02/2024-02-26T12:46:32.302333/PropertyPostCodePollingStationWebLookup-2024-02-26 MID DEVON.TSV"
+    stations_name = "2024-05-02/2024-02-26T12:46:32.302333/PropertyPostCodePollingStationWebLookup-2024-02-26 MID DEVON.TSV"
+    elections = ["2024-05-02"]
     csv_delimiter = "\t"
 
     def station_record_to_dict(self, record):
         # Chawleigh Jubilee Hall, Chawleigh
-        if record.polling_place_id == "10823":
-            record = record._replace(
-                polling_place_easting="", polling_place_northing=""
-            )
+        if record.pollingplaceid == "11109":
+            record = record._replace(pollingplaceeasting="", pollingplacenorthing="")
 
         return super().station_record_to_dict(record)
 
     def address_record_to_dict(self, record):
-        uprn = record.property_urn.strip().lstrip("0")
+        uprn = record.uprn.strip().lstrip("0")
 
         if uprn in [
             "10024639781",  # FOXHOLES FARM, CLAYHIDON, CULLOMPTON
             "10002162361",  # POTTERS, SHILLINGFORD, TIVERTON
-            "10093536511",  # RAMSTORLAND STAG VIEW, STOODLEIGH, TIVERTON
             "200004004367",  # CARAVANS 3 AND 4 THREE ACRES UFFCULME ROAD, WILLAND
             "200004004366",  # CARAVANS 1 AND 2 THREE ACRES UFFCULME ROAD, WILLAND
-            "200004002192",  # PRIMROSE LODGE, LOXBEARE, TIVERTON
             "10009448630",  # MAYFIELD HOUSE, TEMPLETON, TIVERTON
             "10002165647",  # POUND CASTLE, BICKLEIGH, TIVERTON
+            "10095581422",  # OAKLEIGH BARN, BICKLEIGH, TIVERTON
             "200004005422",  # GUNSTONE LODGE GUNSTONE PARK ROAD FROM GUNSTONE CROSS TO GUNSTONE MILL CROSS, GUNSTONE
             "200004006381",  # GUNSTONE PARK, GUNSTONE, CREDITON
-            "10093533623",  # RAVENS WOOD, YEOFORD, CREDITON
             "10093538401",  # COPSTONE HOUSE, BLACK DOG, CREDITON
-        ]:
-            return None
-
-        if record.addressline6 in [
-            # look wrong
-            "EX16 8RA",
```

Comment:
I'm still unconvinced by the station the addresses in `EX16 8RA` are assigned to and think we should exclude.

Reply:
Added in